### ISSUE TITLE
Event options with tick boxes show unknown

### ISSUE THREAD

**Issue by raz3r83** [jamesread/lanlist#10]
Options like Showers, Bring Your Own Alcohol, Smoking Area continue to show as Unknown even if someone ticks then unticks the option.

Can these be changed to Yes/No dropdowns so that they can then be Unknown if not chosen and otherwise answered by the user.

 
![image](https://github.com/jamesread/lanlist/assets/24259311/95412350-654d-4860-a832-11c7d39fe201)

![image](https://github.com/jamesread/lanlist/assets/24259311/eadc834b-7ee4-4738-a25a-c7e67281216a)

**Comment by jamesread** [comment]
Hey @raz3r83 , I just pushed commit af6a2952ccebace5473c1f414586eb4500373b2b to the dev site - it standardizes these event fields in in the edit form, and the view event page, and switches the values on the edit page to "dropdowns. 

Do you think this is better?

Edit page; 

![image](https://github.com/jamesread/lanlist/assets/2025715/0375cf87-8663-4233-a5ef-8fa644b5f1f4)

View page;

![image](https://github.com/jamesread/lanlist/assets/2025715/6691314f-d417-45a5-a1f3-9c4ce11bec36)

You can play with this on the dev site; http://dev.lanlist.info

**Comment by raz3r83** [comment]
yeah looks much better only thing I might still ask to be changed is the Bring your own alcohol? question and instead have that question the same as on the displayed form Alcohol allowed? with the options you have then given.

**Comment by jamesread** [comment]
@raz3r83 , What about making the label "Alcohol policy"?, or just "Alcohol"?

**Comment by raz3r83** [comment]
Either of those would be perfectly fine, I think I would lean more to Alcohol Policy.

**Comment by ilumos** [comment]
I agree changing the field name to "Alcohol" is neater.

In the same way, perhaps change "Smoking area?" to "Smoking"

**Comment by jamesread** [comment]
Renamed to "Alcohol Policy" in PR #13 .
